pybossa/data_access.py

```diff
@@ -155,7 +155,7 @@ def ensure_valid_user_access_levels(access_levels):
 
 @when_data_access()
 def copy_user_data_access_levels(target, access_levels):
-    ensure_valid_user_access_levels(access_levels)
+    #if target:
     target['data_access'] = access_levels
```

Comment:
isn't ensure_valid_user_access_levels check reqd?

Reply:
yes, i must have accidentally deleted this when i was testing something. thank you for catching it!